tests/pubsub/utils.nim

```diff
@@ -201,3 +207,18 @@ proc waitSubGraph*(nodes: seq[PubSub], key: string) {.async.} =
 
     await sleepAsync(5.milliseconds)
     doAssert Moment.now() < timeout, "waitSubGraph timeout!"
+
+proc waitForMesh*(
```

Comment:
This doesn't seem to be used.

Reply:
It was used in a test that was removed by your suggestion. Removing it.